modules/cache-common/src/test/java/org/opensearch/cache/common/tier/MockDiskCache.java

```diff
@@ -79,7 +81,12 @@ public void invalidateAll() {
 
     @Override
     public Iterable<K> keys() {
```

Comment:
Consider swapping to a lambda to avoid the multilayer nesting.

Reply:
Changed.